devices.js

```diff
@@ -2647,6 +2647,30 @@ const devices = [
             await configureReporting.batteryPercentageRemaining(endpoint);
         },
     },
+    {
+        zigbeeModel: ['TRADFRI SHORTCUT Button'],
+        model: 'E1812',
+        vendor: 'IKEA',
+        description: 'TRADFRI Shortcut Button',
+        supports: 'on, brightness up/stop',
+        fromZigbee: [
+            fz.command_on, fz.legacy_genOnOff_cmdOn, fz.command_move,
+            fz.legacy_E1743_brightness_up, fz.command_stop, fz.legacy_E1743_brightness_stop, fz.battery,
+        ],
+        exposes: [e.battery(), e.action(['on', 'brightness_move_up', 'brightness_stop'])],
+        toZigbee: [],
+        // OTA is not supported at the moment
+        // ota: ota.tradfri,
```

Comment:
I think we can enable this as I expect ikea to update their image index soon with this.